routes/api/contacts.js

```diff
@@ -1,25 +1,108 @@
 const express = require('express')
-
+const joi = require('joi')
+const { listContacts,getContactById, addContact, removeContact,updateContact,} = require("../../models/contacts.js");
```

Comment:
```suggestion
const {
 listContacts,
 getContactById, 
 addContact, 
 removeContact,
 updateContact,
 } = require("../../models/contacts.js");
```
Staraj się nie mieć w lini więcje niz 80 znaków. Może prettier by się przydał? :)